### ISSUE TITLE
Triggering Production Step Functions (AWS)

### ISSUE THREAD

**Issue by jdonahue94** [Netflix/metaflow#1602]
Team - I have created a AWS step function with the following command.

`insert_name_here.py  --production step-functions create`

Is there a way to trigger this production step function manually without using the AWS UI?

Thanks,
Jacob

**Comment by savingoyal** [comment]
yep - `insert_name_here.py  --production step-functions trigger`

**Comment by jdonahue94** [comment]
Great. I thought so as well.

Though, the problem I run into is that the above seems to run whatever I have currently saved locally vs what was saved when I ran create.

For example, say I had the following when I ran create:

`param = True`

I then saved the following:

`param = False`

I then ran:

`insert_name_here.py  --production step-functions trigger`

And the production step function ran but with `param = False` rather than `True`.

I was expecting param = True since that was what the production step was created with.

**Comment by savingoyal** [comment]
`trigger` also passes in the parameters to the step-function. you can always do `insert_name_here.py  --production step-functions trigger --param True`

**Comment by jdonahue94** [comment]
Agreed. 

I'm saying this param was hard coded in` insert_name_here.py` and not passed as a param in the cli command.

The production run ran what was most recently saved locally vs what was saved when `create` was called.

That is:

`param = True` was hardcoded and saved locally when create was called.

vs.

`param = False` was hardcoded and saved locally when trigger was called.

My thinking was that `insert_name_here.py  --production step-functions trigger` would run the code that was snapshotted when create was called regardless of what was saved locally.

**Comment by jdonahue94** [comment]
Friendly bump of the above. Thanks.
